.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -14,13 +14,12 @@ If adding a new definition:
 - [ ] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
 - [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
 - [ ] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
-- [ ] `tslint.json` should be present and it shouldn't have any additional or disabling of rules. Just content as `{ "extends": "dtslint/dt.json" }`. If for reason the some rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]`  and not for whole package so that the need for disabling can be reviewed.
+- [ ] `tslint.json` should be present and adhere to [the required form](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson).
```

Comment:
I worry that some (many?) users won't click through so I'd like to include the most important bit here: "no disabling rules".  If someone disagrees, they can click through and see more nuanced guidance.

Reply:
Also, "be present" seems redundant.

Reply:
That's smart, done :heavy_check_mark: